repositories/repositories.go

```diff
@@ -66,9 +69,13 @@ func NewRepositories(
 		CustomListRepository:          &CustomListRepositoryPostgresql{},
 		UploadLogRepository:           &UploadLogRepositoryImpl{},
 		AwsS3Repository:               AwsS3Repository{s3Client: NewS3Client()},
-		GcsRepository:                 &GcsRepositoryImpl{},
+		GcsRepository:                 &gcsRepository,
 		MarbleAnalyticsRepository: MarbleAnalyticsRepository{
 			metabase: metabase,
 		},
+		TransferCheckEnrichmentRepository: NewTransferCheckEnrichmentRepository(
```

Comment:
Now, I'm not 100% happy with this, because we inject a repository (gcsRepository) into another. The issue is that with this, it's harder to do like we do on the ingestion usecase where we allow the option to use a "fake" gcs repository (with local file system) rather than the real GCS repo connecting to GCP - because that should be decided at the usecase level and I don't really want to create a "fakeGcsTransfercherckEnrichmentRepo".
But it's fine for now I guess.